bee-node/Dockerfile

```diff
@@ -0,0 +1,58 @@
+############################
+# Prepare dependencies recipe
+############################
+FROM lukemathwalker/cargo-chef as planner
+
+WORKDIR /src
+COPY . .
+
+RUN cargo chef prepare --recipe-path recipe.json
+
+############################
+# Dependency cache
+############################
+FROM lukemathwalker/cargo-chef as cacher
+
+RUN apt-get update && \
+    apt-get install cmake clang -y
+
+WORKDIR /src
+COPY --from=planner /src/recipe.json recipe.json
```

Comment:
Instead of `/src` I'ld use `/bee`, otherwise it might be confused with the actual `src` folders in the sub-crates.

Reply:
Alright, changing the workdir to `/bee`